game-core/src/main/java/games/strategy/triplea/delegate/GameStepPropertiesHelper.java

```diff
@@ -310,10 +308,8 @@ private static boolean isNonCombatDelegate(final GameData data) {
   }
 
   private static boolean isCombatDelegate(final GameData data) {
-    if (data.getSequence().getStep().getName().endsWith("NonCombatMove")) {
-      return false;
-    }
-    return data.getSequence().getStep().getName().endsWith("CombatMove");
+    return !data.getSequence().getStep().getName().endsWith("NonCombatMove") && data.getSequence().getStep().getName()
```

Comment:
more formatting, please line break before the `&&` operator instead of on the dot operator in `.endsWith`